azure-client-runtime/src/main/java/com/microsoft/azure/v2/Value.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.v2;
+
+/**
+ * A container for a generic type. Serves a similar purpose as pointers in C/C++.
+ * @param <T>
+ */
+class Value<T> {
```

Comment:
Maybe mention that this is a workaround for Java disallowing mutation of local variables in closures. That's basically the usage of it, right?

Reply:
Yeah, that's it. I'll do that.